src/search.c

```diff
@@ -295,7 +297,7 @@ static void print_final_results(bool display_size)
 		/* do not print the size information when the scope is only one bundle ('o')
 		 * because we did not load all bundles and therefore do not have include sizes
 		 * for the result */
-		printf("Bundle %s\t%s", b->bundle_name, b->is_tracked ? "[installed]\t" : "");
+		printf("Bundle %s%s\t%s", b->bundle_name, b->is_experimental ? " (experimental)" : "", b->is_tracked ? "[installed]\t" : "");
```

Comment:
same comment I made on the bundle-list PR. It would be nice to have this as a generic function